tests/test_notation_sets.py

```
@@ -0,0 +1,182 @@
+        "identity": "id"
+    },
+    {
+        "framework": "Order Theory (Posets)",
+        "arrow": "x ≤ y",
+        "composition": "x ≤ y ∧ y ≤ z ⇒ x ≤ z",
+        "identity": "x ≤ x"
+    }
+]
+
+
+def canonical_ir_encode(item_type: str, **params) -> str:
```

Ruff (0.16.2)

[warning] Missing type annotation for `**params`

(ANN003)